app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -69,6 +69,12 @@
     @Persisting(6)
     @MetaField(META_LAST_MODIFIED)
     private Date lastModified;
+
+    /**
+     * Unique id of the CC app to which this form belongs
+     */
+    @Persisting(7)

```

Comment:
Question: This Database is in userspace, which should be app specific. What is this app Id used for?